### PR TITLE
Payment view

### PR DESCRIPTION
**Pull request by Egoes98**
This class has the methods and the view for reservation. This lets you choose how to pay and insert payment data. This view is called once you clicked the confirm button to make a reservation. As iker did it, it conects to the paypal or mastercard servers and there the payment is handled. If payment is accepted the reservation is done.

### CODE REVIEW

**Review by iker-barriocanal — CHANGES_REQUESTED** [review]
Please, document code writing a brief description of what methods do, not just the description of arguments.

File: src/main/java/es/deusto/spq/client/gui/views/guest/UserPayView.java [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+		form.add(lblPassword);
+
+		tFPassword = new JTextField();
+		tFPassword.setBounds(207, 51, 121, 20);
+		tFPassword.setColumns(10);
+		tFPassword.setVisible(false);
+		form.add(tFPassword);
+
+		frame.toFront();
+		frame.setVisible(true);
```
Comment (iapellaniz) [inline src/main/java/es/deusto/spq/client/gui/views/guest/UserPayView.java:234]: Missing call to the `addDisposeEventHandler` method here.

File: src/main/java/es/deusto/spq/client/gui/views/guest/UserPayView.java [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+		 */
+		REQUIRED_FIELD_EMPTY,
+
+		/**
+		 * Payment method not selected
+		 */
+		SELECT_PAYMENT_METHOD,
+
+		/**
+		 *
```
Comment (iapellaniz) [inline src/main/java/es/deusto/spq/client/gui/views/guest/UserPayView.java]: Empty comment

**Review by iapellaniz — COMMENTED** [review]
What data can I use for payment?

**Comment by Egoes98** [comment]
I have added a txt in the last commit with the accounts that can be use for paying, also you can see them in this comment.
PayPal accounts

| Username  | Password |
| ------------- | ------------- |
| iker-barriocanal  | 123  |
| iapellaniz  | deusto  |
| Egoes98 | 7954 |
| Arribas12 | arribas12 |
| dusekvojtech | deusto2019 |

MasterCard accounts

| CardNumber | SecretCode |
| -------------- | ------------- |
|123456789|123|
|465789123|456|
|789123456|789|

**Review by iker-barriocanal — APPROVED** [review]
Good!

**Pull request merged**